src/Bab/RabbitMq/Response.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Bab\RabbitMq;
+
+class Response
```

Comment:
In order to be able to use a different httpClient in the future. To not be coupled with Guzzle object.